internal/otelcollector/config/metric/gateway/service.go

```diff
@@ -19,7 +19,7 @@ func makeInputPipelineServiceConfig(pipeline *telemetryv1alpha1.MetricPipeline)
 func makeAttributesEnrichmentPipelineServiceConfig(pipelineName string) config.Pipeline {
 	return config.Pipeline{
 		Receivers:  []string{formatRoutingConnectorID(pipelineName)},
-		Processors: []string{"k8sattributes", "transform/resolve-service-name", "resource/drop-kyma-attributes"},
+		Processors: []string{"k8sattributes", "transform/resolve-service-name"},
```

Comment:
why has this been changed

Reply:
because we have now some new attribute `kyma.input.name` added by the agent, in this stage the `drop-kyma-attributes` will drop the attrbiute before reach the other components requires the attribute added by the agent.

I moved `drop-kyma-attributes` to the end of the component chain